homework6/index.html

```diff
@@ -0,0 +1,573 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>Homework6</title>
+  <link rel="stylesheet" href="./style.css">
+</head>
+<body>
+  <div class="main-container">
+    <!--- HEADER -->
+    <header class="header">
+      <div class="header-panel">
+        <span class="header-telephone">
+          <img src="./img/telephone.png">(044)123-45-67 (044)567-89-10
+        </span>
+        <span class="authorize">
+          <div class="authorize-logo">
+            <div class="authorize-logo-up"></div>
+            <div class="authorize-logo-down"></div>
+          </div>
+          <input class="authorize-input" type="password" placeholder="логин">
+          <input class="authorize-input" type="password" placeholder="пароль">
+          <a class="authorize-button" href="http://google.com">вход</a>
+          <span class="authorize-text">|</span>
+          <a class="authorize-button" href="http://google.com">регистрация</a>
+          <a class="authorize-button lost-parol" href="http://google.com">забыли пароль?</a>
+        </span>
+        <nav class="navigation">
+          <span class="navigation-button"><a href="http://google.com"><img src="./img/nav_home.png"></a></span>
+          <span class="navigation-button"><a href="http://google.com"><img src="./img/nav_map.png"></a></span>
+          <span class="navigation-button"><a href="http://google.com"><img src="./img/nav_mail.png"></a></span>
+        </nav>
+      </div>
+      <div class="header-img"><img src="./img/header_img.png"></div>
+      <nav class="header-menu">
+        <a class="menu-button" href="http://google.com">о компании</a>
+        <a class="menu-button" href="http://google.com">оплата</a>
+        <a class="menu-button" href="http://google.com">доставка</a>
+        <a class="menu-button" href="http://google.com">статьи</a>
+        <a class="menu-button" href="http://google.com">контакты</a>
+        <div class="basket">
+          <div class="basket-img"><img src="./img/basket.png"></div>
+          <span class="basket-text">товаров:</span>
+          <span class="basket-total-ware">0</span>
+          <span class="basket-text total-cash">на сумму:</span>
+          <span class="basket-total-cash">0,00</span>
+          <span class="basket-currency">грн.</span>
+        </div>
+      </nav>
+    </header>
```

Comment:
![image](https://user-images.githubusercontent.com/47972692/133886382-baeb4ab6-d2b5-4f40-8a68-4e37549f6a7f.png)

Reply:
In Chrome and Opera I can`t have this problem. Scroll appear when I use resize .